readme_es.md

```diff
@@ -0,0 +1,219 @@
+    </a>
+    <a href="https://github.com/anuraghazra/github-readme-stats/issues">
+      <img alt="Issues" src="https://img.shields.io/github/issues/anuraghazra/github-readme-stats?color=0088ff" />
+    </a>
+    <a href="https://github.com/anuraghazra/github-readme-stats/pulls">
+      <img alt="GitHub pull requests" src="https://img.shields.io/github/issues-pr/anuraghazra/github-readme-stats?color=0088ff" />
+    </a>
+  </p>
+
+  <p align="center">
+    <a href="#ejemplo">Ve un ejemplo</a>
+    ·
+    <a href="https://github.com/anuraghazra/github-readme-stats/issues">Reporta un bug</a>
+    ·
+    <a href="https://github.com/anuraghazra/github-readme-stats/issues">Solicita una mejora</a>
+  </p>
+</p>
+<p align="center">¿Te gusta este proyecto? ¡Por favor considera <a href="https://www.paypal.me/anuraghazra">donar</a> para ayudar a mejorarlo!
+
+# Características
+
+- [Tarjeta de estadísticas de GitHub](#tarjeta-de-estadísticas-de-github)
+- [Pins extra de GitHub](#pins-extra-de-github)
+- [Temas](#temas)
+- [Customización](#customización)
+- [Despliega por tu cuenta](#despliega-tu-propia-instancia-de-vercel)
+
+# Tarjeta de estadísticas de GitHub
+
+Copia y pega esto en el contenido de tu README.md y listo. ¡Simple!
+
+Cambia el valor `?username=` al nombre de tu usuario de GitHub.
+
+```md
+[![Anurag's github stats](https://github-readme-stats.vercel.app/api?username=anuraghazra)](https://github.com/anuraghazra/github-readme-stats)
+```
+
+_Nota: las clasificaciones se calculan basándose en las estadísticas del usuario. Ve [src/calculateRank.js](./src/calculateRank.js)._
+
+### Ocultar estadísticas individualmente
+
+Para ocultar alguna estadística específica, puedes utilizar el parámetro `?hide=` con un arreglo de items que quieras ocultar.
+
+> Opciones: `&hide=["stars","commits","prs","issues","contribs"]`
+
+```md
+![Anurag's github stats](https://github-readme-stats.vercel.app/api?username=anuraghazra&hide=["contribs","prs"])
+```
+
+### Mostrar íconos
+
+Para habilitar los íconos, puedes utilizar `show_icons=true` como parámetro, de esta manera:
+
+```md
+![Anurag's github stats](https://github-readme-stats.vercel.app/api?username=anuraghazra&show_icons=true)
+```
+
+### Temas
+
+Puedes customizar el aspecto de la tarjeta sin realizar ninguna [customización manual](#customización) con los temas incorporados.
+
+Utiliza el parámetro `?theme=THEME_NAME`, de esta manera:
+
+```md
+![Anurag's github stats](https://github-readme-stats.vercel.app/api?username=anuraghazra&show_icons=true&theme=radical)
+```
+
+#### Todos los temas incorporados:
+
+dark, radical, merko, gruvbox, tokyonight, onedark, cobalt, synthwave, highcontrast, dracula
+
+<img src="https://res.cloudinary.com/anuraghazra/image/upload/v1595174536/grs-themes_l4ynja.png" alt="Github Readme Stat Themes" width="600px"/>
+
+Descubre más temas en el [archivo de configuración](./themes/index.js). También, **puedes contribuir añadiendo más temas** si lo deseas :D
```

Comment:
I added a theme preview file earlier, we can add a link to it here.

I would replace that with: 
```diff

- Descubre más temas en el [archivo de configuración](./themes/index.js). También, **puedes contribuir añadiendo más temas** si lo deseas :D
+ Puedes ver una vista previa de [todos los temas disponibles](./themes/README.md) o ver el [archivo de configuración](./themes/index.js) del tema y también **puedes contribuir con nuevos temas** si lo deseas: D 
```